pyproject.toml

```diff
@@ -1,3 +1,142 @@
+# PACKAGING
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+dynamic = ["version"]
+name = "python-telegram-bot"
+description = "We have made you a wrapper you can't refuse"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = "GPL-3.0-only"
```

Comment:
isn't this different from `setup.py`'s LGPLv3?

Reply:
used `license-files` instead, now.